tstest/natlab/vnet/vnet_test.go

```diff
@@ -354,6 +370,40 @@ func mkDHCP(srcMAC MAC, typ layers.DHCPMsgType) []byte {
 	return mkPacket(eth, ip, udp, dhcp)
 }
 
+func mkSyslogPacket(srcIP netip.Addr, msg string) []byte {
+	eth := &layers.Ethernet{
+		SrcMAC:       nodeMac(1).HWAddr(),
+		DstMAC:       routerMac(1).HWAddr(),
+		EthernetType: layers.EthernetTypeIPv4,
+	}
+	if srcIP.Is6() {
+		eth.EthernetType = layers.EthernetTypeIPv6
+	}
+	var ip serializableNetworkLayer
+	if srcIP.Is4() {
```

Comment:
probably could make a helper for v4/v6 creation

Reply:
will do (separately, next)

Reply:
done, in https://github.com/tailscale/tailscale/pull/13329